web-common/src/features/dashboards/leaderboard/leaderboard-utils.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { makeHref } from "@rilldata/web-common/features/dashboards/leaderboard/leaderboard-utils";
+import { describe, it, expect } from "vitest";
+
+describe("makeHref", () => {
+  const TestCases: [
+    string | number | boolean | null,
+    string,
+    string | undefined,
+  ][] = [
+    [null, "http://localhost/dim", undefined],
+    ["http://localhost", "http://localhost/dim", "http://localhost"],
+    [true, "http://localhost/dim", "http://localhost/dim"],
+    [1, "http://localhost/dim", "http://localhost/dim"],
+  ];
```

Comment:
The test cases would be more readable if we used a descriptive object rather than a raw tuple:
```
const testCases = [
    { uri: null, dimValue: "http://localhost/dim", expected: undefined },
    { uri: "http://localhost", dimValue: "http://localhost/dim", expected: "http://localhost" },
    { uri: true, dimValue: "http://localhost/dim", expected: "http://localhost/dim" },
    { uri: 1, dimValue: "http://localhost/dim", expected: "http://localhost/dim" },
  ];
```